.github/workflows/Build.yml

```diff
@@ -1,6 +1,8 @@
 name: Build
 
 on:
+  schedule:
+    - cron: "42 6 * * 1-5"
```

Comment:
Mon-Fri 6:42 UTC

The 42 is to avoid GH rush hour. [Docs](https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#schedule) say:

> The schedule event can be delayed during periods of high loads of GitHub Actions workflow runs. High load times include the start of every hour. If the load is sufficiently high enough, some queued jobs may be dropped. To decrease the chance of delay, schedule your workflow to run at a different time of the hour.